moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp

```diff
@@ -61,7 +60,6 @@ MoveItCpp::MoveItCpp(const rclcpp::Node::SharedPtr& node, const Options& options
 {
   if (!tf_buffer_)
     tf_buffer_ = std::make_shared<tf2_ros::Buffer>(node_->get_clock());
-  tf_listener_ = std::make_shared<tf2_ros::TransformListener>(*tf_buffer_);
```

Comment:
Here the same, we should run a TransformListener independent of the CSM.

Reply:
Same with this comment.

Reply:
See my previous comment